### PR TITLE
Periodic export job

### PR DESCRIPTION
**Pull request by folix-01**

### CODE REVIEW

File: src/collective/volto/formsupport/scripts/send_exported_forms.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+from zope.component import queryMultiAdapter
+from zope.globalrequest import getRequest
+
+import click
+import csv
+import sys
+import transaction
+
+
+LAST_SENDING_DATE_KEY = "collvective.volto.formsupport.LAST_SENDING_DATE"
```
Comment (mamico) [inline src/collective/volto/formsupport/scripts/send_exported_forms.py:25]: Typo

File: src/collective/volto/formsupport/scripts/send_exported_forms.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+            'attachment; filename="%s"' % mail_attachment_filename
+        )
+
+        for recipient in recipients:
+            if not recipient:
+                continue
+            msg = MIMEMultipart()
+            msg.attach(
+                MIMEText(
+                    api.portal.translate(
```
Comment (mamico) [inline src/collective/volto/formsupport/scripts/send_exported_forms.py:89]: Do you have checked if translations really works? We are in a script, don't take it for granted.

File: src/collective/volto/formsupport/scripts/send_exported_forms.py [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+    ),
+)
+@click.option(
+    "--dryrun/--no-dryrun",
+    is_flag=True,
+    default=True,
+    help="--dryrun (default) simulate, --no-dryrun actually save the changes",
+)
+def main(dryrun):
+    def send_csv_email(context, csv_file, recipients):
```
Comment (mamico) [inline src/collective/volto/formsupport/scripts/send_exported_forms.py:44]: Skip sending email if recipients is empty

File: src/collective/volto/formsupport/locales/collective.volto.formsupport.pot [hunk 4]
```diff
@@ -97,6 +101,10 @@ msgstr ""
 msgid "Your code to validate the email address:"
 msgstr ""
 
+#: collective/volto/formsupport/scripts/send_exported_forms.py:93
+msgid "Your forms export from {url} , until the {date}"
```
Comment (mamico) [inline src/collective/volto/formsupport/locales/collective.volto.formsupport.pot:105]:
```suggestion
msgid "Your forms export from {url}, until the {date}"
```

File: src/collective/volto/formsupport/scripts/send_exported_forms.py (hunk 2 above)
Comment (folix-01) [reply src/collective/volto/formsupport/scripts/send_exported_forms.py:89]: works, tested

File: src/collective/volto/formsupport/locales/collective.volto.formsupport.pot (hunk 4 above)
Comment (folix-01) [reply src/collective/volto/formsupport/locales/collective.volto.formsupport.pot:105]: jus wanted to be sure ',' will not be considered as a part of link in the email interpreter

**Pull request closed without merging**